tests/integration_test/integration_test.sh

```diff
@@ -569,20 +569,16 @@ function send_ss2_notification {
 
     print_style "info" "Awaiting workflow completion"
 
+    # Wait for status to update in Cromwell before polling
+    sleep 10
+
     # Uses the docker image built from Dockerfile next to this script
```

Comment:
this comment out of date now? same below for TENX

Reply:
You're right! I'll take that out